monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
+== About {product-title} Container Platform monitoring
+
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects]. {product-title} delivers monitoring best practices out of the box. A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the {product-title} web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster.
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
```

Comment:
At the moment, this line and the preceding one make up one paragraph. You could break them up into multiple paragraphs to help readers scan the content. For example, you could have:

* An initial paragraph that introduces the stack and the differentiation between core monitoring and monitoring for user-defined projects
* A second paragraph that introduces alerts, dashboards and metrics
* A third paragraph that introduces the third party UIs

Reply:
You could potentially remove the "You can use the **Administrator** or ** Developer** perspective" aspect of this line to help minimise the content.

Reply:
+1